### PR TITLE
Update ITEM_en.tsv

### PR DESCRIPTION
**Pull request by emailboxu**
~3416.

### CODE REVIEW

File: ITEM_en.tsv [hunk 1]
```diff
@@ -3373,47 +3373,47 @@ ITEM_20150317_003543	Insect Secretor
...
-ITEM_20150317_003561	Necklace looks shabby
-ITEM_20150317_003562	My love..to Ruklys..from Lidia. has written.
-ITEM_20150317_003563	Skirt of Kid Demons
-ITEM_20150317_003564	A skirt obtained to defeat both Pawndel and Pawnd. It is a material to make the demon transformation scroll.
+ITEM_20150317_003553	An ingredient necessary to make the Holy Bomb. The purified result of Wicked Essence.
+ITEM_20150317_003554	Romuba Holy Water 
+ITEM_20150317_003555	When consumed it will prevent Demons from detecting your presence!
+ITEM_20150317_003556	Holy Bomb
+ITEM_20150317_003557	A bomb blessed with holy power. It can defeat Glizardon in a single blow.
+ITEM_20150317_003558	Holy Jewel
```
Comment (Sunaries) [inline ITEM_en.tsv:3387]: Holy Spirit Stone is good. You omitted 'Spirit'.

File: ITEM_en.tsv [hunk 2]
```diff
@@ -3373,47 +3373,47 @@ ITEM_20150317_003543	Insect Secretor
...
-ITEM_20150317_003581	Material of crude exploration group to find the first part of stone statue. It needs to be processed with enough amount.
-ITEM_20150317_003582	Crude exploration bar
-ITEM_20150317_003583	Exploration bar that to find the first part of tombstone which has been hidden in somewhere.
-ITEM_20150317_003584	Part carved with dust
-ITEM_20150317_003585	Part that epigraphist Raymond though it is second part of tombstone. We don't know whether there are second part is placed or not.
-ITEM_20150317_003586	Sap of short tree
-ITEM_20150317_003587	Sap of short tree asked by solicitor Edang. It has high value as an effect to cure.
+ITEM_20150317_003574	Use this scroll to transform to Panwndel or Pawnd.
+ITEM_20150317_003575	Light Crystal
+ITEM_20150317_003576	A Power Crystal processed and filled with the full strength of Light. It can release the enchatnment that binds the sealing the Cathedral doors.
```
Comment (Sunaries) [inline ITEM_en.tsv:3405]: Isn't it better to decapitalize Light?

**Comment by imcgames** [comment]
Thank you :+1: 
We retained Holy Spirit Bomb and Holy Spirit Stone as is for now :smile:

File: ITEM_en.tsv (hunk 2 above)
Comment (emailboxu) [inline ITEM_en.tsv:3405]: Dear God I missed so many grammar/spelling errors, will fix this in another PR.

**Pull request merged**